pkg/cmd/server/apis/config/v1/types.go

```diff
@@ -247,6 +247,7 @@ type MasterConfig struct {
 	EtcdConfig *EtcdConfig `json:"etcdConfig"`
 	// OAuthConfig, if present start the /oauth endpoint in this process
 	OAuthConfig *OAuthConfig `json:"oauthConfig"`
+
```

Comment:
same as above

Reply:
yeah remamnt of a previous incarnation, will remove, thanks for catching